builds.yml

```diff
@@ -27,6 +27,7 @@ buildTypes:
       - REQUIRE_SNAPS_ALLOWLIST: true
       - IFRAME_EXECUTION_ENVIRONMENT_URL: https://execution.consensys.io/3.0.0/index.html
       - KEYRING_SNAPS_REGISTRY_URL: https://metamask.github.io/keyring-snaps-registry/prod/registry.json
+      - KEYRING_SNAPS_DATE: 02 Nov 2023 15:00:00 GMT
```

Comment:
I think we should use a more specific date, e.g. `KEYRING_SNAPS_AVAILABILITY_DATE`, otherwise we'll be asking "which date?".

Reply:
Solved in [c265057](https://github.com/MetaMask/metamask-extension/pull/21221/commits/c265057fb8789f849932287742df4d7734174e69)